documentation/builders/event-devices.md

```diff
@@ -0,0 +1,134 @@
+# Event devices
+
+## Background
+Event devices are generic input devices that are exposed in `/dev/input`.
+This includes USB peripherals (Keyboards, Controllers, Joysticks or Mouse) as well as potentially bluetooth devices.
+
+A specific usecase for this could be, if a Zero Delay Arcade USB Encoder is used to wire arcade buttons instead of using GPIO pins.
+
+This functionality was previously implemented under the name of [USB buttons](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/components/controls/buttons_usb_encoder/README.md?plain=1).
+
+The devices and their button mappings need to be mapped in the configuration file.
+
+## Configuration
+
+To configure event devices, first add the plugin as the last entry to the module list:
+
+``` yaml
+  modules:
+    named:
+      event_devices: controls.event_devices # Add as last entry
+```
+
+Then add the following section to the configuration:
```

Comment:
Similar to [gpio.yaml](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/resources/default-settings/gpio.example.yaml), I recommend to outsource this to it's own file, e.g. `evdev.yaml`. You could then also proovide an `evdev.example.yaml`